test/mesh/test_domain.jl

```diff
@@ -46,4 +46,210 @@
         @test "ymin" ∉ values(boundary_names(new_mesh))
         @test Bcube.boundary_faces(new_mesh, "ymax") == [7, 10]
     end
+
+    @testset "subdomains" begin
+        @testset "CellDomain" begin
+            mesh = read_mesh(
+                joinpath(
+                    @__DIR__,
+                    "..",
+                    "assets",
+                    "rectangle-mesh-tri-quad-nx10-ny10.msh22",
+                );
+                warn = false,
+            )
+            Ω = CellDomain(mesh)
+            subdomains = Bcube.get_subdomains(Ω)
+            @test length(subdomains) == 2
+            @test sum(sdom -> length(Bcube.get_indices(sdom)), subdomains) == ncells(mesh)
+            # subdomain #1
```

Comment:
could you please add a comment specifying how you initially checked those numbers? In gmsh?

Reply:
done